tests/SideBySide.New/QueryTests.cs

```diff
@@ -128,17 +128,9 @@ public void NextResultBeforeRead()
 
 					Assert.Throws<MySqlException>(() => cmd2.ExecuteReader());
 					Assert.Throws<MySqlException>(() => cmd2.ExecuteScalar());
-
-#if NET45
```

Comment:
Can this be a `#if !NETSTANDARD1_3` (or `#if NET451`?), too? (I assume the `.Close()` method is still available there (and that it's worth testing that it fully closes an open reader).